bin/dartdoc.dart

```diff
@@ -103,8 +100,6 @@ ArgParser _createArgsParser() {
       help: "Location of the Dart SDK. Use if SDK isn't automatically located.");
   parser.addFlag('sdk-docs',
       help: 'Generate ONLY the docs for the Dart SDK.', negatable: false);
-  parser.addOption('sdk-readme',
```

Comment:
Would like to keep this around a little while longer. The stable build of the SDK still does not have the readme included.